amphibia-reptilia.csl

```diff
@@ -58,13 +63,13 @@
       </group>
     </layout>
   </citation>
-  <bibliography et-al-min="7" et-al-use-first="6">
+  <bibliography delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all" entry-spacing="2" line-spacing="2" hanging-indent="true">
     <sort>
       <key macro="author"/>
       <key macro="issued"/>
       <key variable="title"/>
     </sort>
-    <layout suffix=".">
+    <layout prefix="•&#9;" suffix=".">
```

Comment:
Bedankt Rintze. I hadn't seen a PDF yet, good to know, because it was not clear from the Instructions to authors. I'll change it as well, and I'll use the PDF to find definitions for the types that are not defined in the instructions (e.g. edited book).